xml/System.IO.Enumeration/FileSystemEnumerator`1.xml

```diff
@@ -227,8 +227,8 @@
         <Parameter Name="directory" Type="System.ReadOnlySpan&lt;System.Char&gt;" />
       </Parameters>
       <Docs>
-        <param name="directory">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="directory">The path of the directory that finished.</param>
+        <summary>Called whenever the end of a directory is reached.</summary>
```

Comment:
The *On* methods are usually reserved to raise events. Is that the case here?

Reply:
Similar to the case above, I'll use an intro like "When overriden in a derived class...". The class' behavior is summarized here: https://blogs.msdn.microsoft.com/jeremykuhne/2018/03/09/custom-directory-enumeration-in-net-core-2-1/